### PR TITLE
Interpolate 11 exposed to Python

### PR DESCRIPTION
**Pull request by tomdol**
### Details:
 - Interpolate v11 exposed to Python

### Tickets:
 - 104324

### CODE REVIEW

**Review by p-wysocki — APPROVED** [review]

File: src/bindings/python/tests_compatibility/test_ngraph/test_create_op.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 import ngraph.opset1 as ng_opset1
 import ngraph.opset5 as ng_opset5
 import ngraph.opset10 as ng_opset10
```
[the remark this answers is not in the document]
Comment (tomdol) [reply src/bindings/python/tests_compatibility/test_ngraph/test_create_op.py:13]: This file still tests the previous version of the Interpolate op (`def test_interpolate_opset10`) so I think both imports are required.

**Comment by akuporos** [comment]
Please,
Also add information about new `opset11` to these files:
* [Documentation](https://github.com/openvinotoolkit/openvino/blob/master/docs/api/ie_python_api/api.rst)
* [setup.py](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/setup.py#L57)
* [node_factory.py](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/openvino/runtime/utils/node_factory.py#L16)

Shouldn't `opset10` which is used in tests be changed to `opset11`?

CC: @jiwaszki

**Comment by jiwaszki** [comment]
@tomdol @akuporos Yes, generally all opset10 "basic usage" should be switched to opset11 with this PR. If some very specific op from very specific opset needs to be tested - OK, keep it. However bumping version should be tested somehow besides just `interpolate` check.

**Review by jiwaszki — APPROVED** [review]
👏

**Review by akuporos — APPROVED** [review]

**Pull request merged**